docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -850,6 +850,70 @@ sycl_reduce                                       any SYCL      Reduction in a S
+omp_multi_reduce                                              any OpenMP    OpenMP parallel multi-reduction.
+                                                              policy
+omp_multi_reduce_ordered                                      any OpenMP    OpenMP parallel multi-reduction with result
+                                                              policy        guaranteed to be reproducible.
+cuda/hip_multi_reduce_atomic                                  any CUDA/HIP  Parallel multi-reduction in a CUDA/HIP kernel.
+                                                              policy        Multi-reduction may use atomic operations
+                                                                            leading to run to run variability in the
+                                                                            results.
+                                                                            (device synchronization will occur when
+                                                                            reduction value is finalized)
+cuda/hip_multi_reduce_atomic_low_performance_low_overhead     any CUDA/HIP  Same as above, but multi-reduction uses
+                                                              policy        a low overhead algorithm with a minimal
+                                                                            set of resources. This minimally effects
+                                                                            the performance of loops containing the
+                                                                            multi-reducer though it may cause the
+                                                                            multi-reducer itself to perform poorly if
+                                                                            it is used.
+cuda/hip_multi_reduce_atomic_block_then_atomic_grid_host_init any CUDA/HIP  The multi-reduction uses atomics into shared
+                                                              policy        memory and global memory. Atomics into
+                                                                            shared memory are used each time a value
+                                                                            is combined into the multi-reducer and at
+                                                                            the end of the life of the block the shared
+                                                                            values are combined into global memory with
+                                                                            atomics.
+                                                                            The memory for global atomics is
+                                                                            initialized on the host.
```

Comment:
```suggestion
cuda/hip_multi_reduce_atomic_block_then_atomic_grid_host_init any CUDA/HIP  The multi-reduction uses atomics into shared
                                                              policy        memory and global memory. Atomics into
                                                                            shared memory are used each time a value
                                                                            is combined into the multi-reducer and at
                                                                            the end of the life of the block the shared
                                                                            values are combined into global memory with
                                                                            atomics. If there is not enough shared memory
                                                                            available this will fall back to using atomics into
                                                                            global memory only, which may have a
                                                                            performance penalty.
                                                                            The memory for global atomics is
                                                                            initialized on the host.
```